crates/rover-client/src/operations/cloud/config/update.rs

```diff
@@ -90,7 +90,60 @@ mod tests {
     }
 
     #[test]
-    fn null_upsert_router_config_error() {
+    fn test_build_response_errs_with_no_graph() {
```

Comment:
Should we start moving those to table tests using `rstest`?

Reply:
If you have a nice suggestion/example, I'm all ears!